scripts/test-upgrade.sh

```diff
@@ -129,4 +129,7 @@ bash scripts/deploy-rpc-playbooks.sh
 #              For a detailed description, please see the README in
 #              the rpc_post_upgrade role directory.
 cd ${RPCD_DIR}/playbooks
+openstack-ansible -e 'logging_upgrade=true' logstash.yml
```

Comment:
Yeah, it definitely needs to run the upgrade steps of ES and LS before running the vanilla playbooks, so either A or B work.  I'm fine with you submitting a commit.